### ISSUE TITLE
bm03 cannot start/stop agents

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/DigitalMammographyChallenge#551]
```
 docker ps
CONTAINER ID        IMAGE                                                                                                                 COMMAND                  CREATED             STATUS              PORTS               NAMES
3055b05cf7c3        docker.synapse.org/syn8096724/inference-mlo@sha256:5b2f7abcb1df4327da5df22e15963490d8d6634ebe0a41d90ab44f30643cbff7   "/sc1_infer.sh"          32 hours ago        Up 32 hours                             8101074.bm03-2.SCORE_SC_1.SCORING
76469efc8876        docker.synapse.org/syn5644795/nvml-statsd                                                                             "/bin/sh -c 'python /"   4 weeks ago         Up 3 days                               nvml-statsd
29bd0bc53aa2        sysdig/agent                                                                                                          "/docker-entrypoint.s"   4 weeks ago         Up 3 days                               sysdig-agent
ubuntu@ip-172-31-59-223:~/DigitalMammographyChallenge$ docker ps -a
CONTAINER ID        IMAGE                                                                                                                 COMMAND                  CREATED             STATUS                    PORTS               NAMES
c9038be7c631        docker.synapse.org/syn8031514/inference@sha256:34b55c2b7d49c4e1782d13020fb52391f528d3979eac983645b026c5e14b9dbd       "/sc1_infer.sh"          14 hours ago        Exited (0) 4 hours ago                        8111197.bm03-1.SCORE_SC_1.SCORING
```

Disk usage looks OK:
```
[dreamuser@bm03 ~]$ df -h
Filesystem                 Size  Used Avail Use% Mounted on
/dev/sda3                   99G   43G   57G  43% /
devtmpfs                   252G     0  252G   0% /dev
tmpfs                      252G     0  252G   0% /dev/shm
tmpfs                      252G  418M  252G   1% /run
tmpfs                      252G     0  252G   0% /sys/fs/cgroup
/dev/mapper/data-dataset3   10T   73M  9.5T   1% /data/dataset3
/dev/mapper/data-dataset4   10T   73M  9.5T   1% /data/dataset4
/dev/sda1                  248M  210M   39M  85% /boot
/dev/mapper/data-dataset2   10T   73M  9.5T   1% /data/dataset2
/dev/mapper/data-dataset0   10T   73M  9.5T   1% /data/dataset0
/dev/mapper/data-dataset1   10T   73M  9.5T   1% /data/dataset1
/dev/mapper/data-model4    976M  2.6M  907M   1% /data/model4
/dev/mapper/data-scratch1  197G   61M  187G   1% /data/scratch1
/dev/mapper/data-model3    976M  2.6M  907M   1% /data/model3
/dev/mapper/data-model2    976M  2.5M  907M   1% /data/model2
/dev/mapper/data-scratch0  197G   61M  187G   1% /data/scratch0
/dev/mapper/data-data      8.0T  4.2T  3.4T  56% /data/data
/dev/mapper/data-model1    2.0G  3.0M  1.9G   1% /data/model1
/dev/mapper/data-model0    2.0G  267M  1.6G  15% /data/model0
tmpfs                       51G     0   51G   0% /run/user/1000
```

@tschaffter Since this is the one machine where 'sysdig' is running could you have a look and say if anything looks odd?

**Comment by brucehoff** [comment]
Note:  @tschaffter points out he can log in without delay.

**Comment by brucehoff** [comment]
To take care of the current submissions: 
8111197 -- actually finished without error and its predictions were uploaded
8101074 -- is still running and in a healthy state. It's been up for nearly two days.

I manually linked the uploaded results file for 8111197 (syn8115391) to the submission and advanced it to "ACCEPTED" to trigger scoring.  I verified that it was scored.
Although I ultimately want to do a hard reboot on bm03 I would like to let 8101074 finish running first.
